src/bitmap/container.rs

```diff
@@ -94,6 +94,37 @@ impl Container {
         result
     }
 
+    pub fn remove_first(&mut self, n: usize) {
+        match &self.store {
+            Store::Bitmap(bits) => {
+                if bits.len() - n as u64 <= ARRAY_LIMIT {
+                    let mut replace_array = Vec::with_capacity(bits.len() as usize - n);
+                    replace_array.extend(bits.clone().into_iter().skip(n));
```

Comment:
This doesn't need to clone the whole bits container (making a whole new allocation), this should be `bits.iter().skip(n)`

Reply:
fixed
https://github.com/RoaringBitmap/roaring-rs/pull/253/commits/521f5a7a908195177cb7dcabaae277ef58263662